pkg/model/sdk_api.go

```diff
@@ -86,6 +86,24 @@ func (a *SDKAPI) GetOperationMap(cfg *ackgenconfig.Config) *OperationMap {
 			opMap[opType][resName] = op
 		}
 	}
+
+	// We need to do a second pass over the operation overrides because some
+	// APIs have multiple operations of a particular OpType and we need to
+	// always be sure that the overridden operation is the one that we use
+	// during inference.
```

Comment:
Consider citing here a motivating example or simply include a reference to https://github.com/aws-controllers-k8s/community/issues/1555

Reply:
done! :)